### PR TITLE
Fixed #34232 -- Fixed typo in docs/intro/tutorial07.txt.

### PR DESCRIPTION
**Pull request by rajdesai24**
fixed 'Add Another' casing typo in tutorial 7

### CODE REVIEW

File: docs/intro/tutorial07.txt [hunk 1]
```diff
@@ -106,7 +106,7 @@ database. Django knows that a :class:`~django.db.models.ForeignKey` should be
 represented in the admin as a ``<select>`` box. In our case, only one question
 exists at this point.
 
-Also note the "Add Another" link next to "Question." Every object with a
+Also note the "Add another" link next to "Question." Every object with a
```
Comment (felixxm) [inline docs/intro/tutorial07.txt:109]:
```suggestion
Also note the "Add another question" link next to "Question." Every object with a
```

Comment (felixxm) [reply docs/intro/tutorial07.txt:109]: This should also be fixed in line 111:
```diff
diff --git a/docs/intro/tutorial07.txt b/docs/intro/tutorial07.txt
index 9e4279e89f..f7f523f086 100644
--- a/docs/intro/tutorial07.txt
+++ b/docs/intro/tutorial07.txt
@@ -106,12 +106,12 @@ database. Django knows that a :class:`~django.db.models.ForeignKey` should be
 represented in the admin as a ``<select>`` box. In our case, only one question
 exists at this point.
 
-Also note the "Add Another" link next to "Question." Every object with a
-``ForeignKey`` relationship to another gets this for free. When you click "Add
-Another", you'll get a popup window with the "Add question" form. If you add a question
-in that window and click "Save", Django will save the question to the database and
-dynamically add it as the selected choice on the "Add choice" form you're
-looking at.
+Also note the "Add another question" link next to "Question." Every object with
+a ``ForeignKey`` relationship to another gets this for free. When you click
+"Add another question", you'll get a popup window with the "Add question" form.
+If you add a question in that window and click "Save", Django will save the
+question to the database and dynamically add it as the selected choice on the
+"Add choice" form you're looking at.
 
 But, really, this is an inefficient way of adding ``Choice`` objects to the system.
 It'd be better if you could add a bunch of Choices directly when you create the
```

**Comment by rajdesai24** [comment]
hey, @felixxm thank you for the quick review. Added the new changes suggested.

**Review by felixxm — COMMENTED** [review]
@rajdesai24 Thanks :+1: Welcome aboard :boat: 

I fixed the second "Add Another" and wrapped at 79 chars.

**Comment by rajdesai24** [comment]
Thank you @felixxm

**Pull request merged**